### PR TITLE
feature (SELC-1498): migrate to auth0

### PR DESCRIPTION
**Pull request by dluschitsky**
This PR addresses issue https://integrate.atlassian.net/browse/SELC-1498

### CODE REVIEW

**Review by quetzyg — APPROVED** [review]
Left a minor note, not blocking

File: src/Client.php [hunk 1]
```diff
@@ -37,7 +37,9 @@ class Client
      * @var array
      */
     protected $defaults = [
-        'endpoint' => 'https://api.integrate-events.com/v2',
+        'endpoint' => 'https://api.integrate-events.com/v2.1',
+        'auth_endpoint' => 'https://integrate-unified-dev.auth0.com/oauth/token',
```
[the remark this answers is not in the document]
Comment (sablake) [reply src/Client.php:41]: This needs to be the production domain. Ideally https://auth.integrate.com/. If it's not ready by the 9th, then use https://integrate-inc.auth0.com/ instead

Wait for @sriramgopaldevops & @quetzyg to confirm.

**Review by sablake — CHANGES_REQUESTED** [review]

**Comment by dluschitsky** [comment]
@quetzyg @sablake the Auth domain was changed to https://auth.integrate.com/.

**Comment by quetzyg** [comment]
Looks good

**Review by sablake — APPROVED** [review]

**Pull request merged**